docs/ww3_doxy_tmpl.md

```diff
@@ -0,0 +1,68 @@
+# WAVEWATCH III Doxygen Header Reference
+<br>
+
+_Doxygen markup headers to be placed directly above the respective Fortran code units._
+
+### FILE
+```
+!> @file  <file name>
+!> @brief  <one sentence description>
+!> 
+!> @author <author name> @date <dd-Mon-yyyy>
+```
+
+### MODULE
+```
+!> @brief <one sentence description>   # contents of _1. Purpose_ from original header
+!> 
+!> @details <extended description>
```

Comment:
The details tag is never needed and should not be used.

Doxygen is set up to allow a lot of different ways to specify the docs. It's possible, for example, to have the docs completely separate from the code files. And in this case, some of these extra tags may be needed.

But in our case, we are using doxygen in the simplest and most programmer friendly way. Whatever is not needed, should not be asked for, giving programmers the easiest and simplest path to good documentation habits.

So the details tag should be removed everywhere in this document. It's not needed.

Reply:
I will leave this call up to the code managers.  I had left these in because for me they gave some visual order when scanning through the code.  I do acknowledge they're functionally unnecessary though, so that case is understood.  I'm fine either way.

Reply:
I mulled the options and decided to leave these in.  I gave my thinking in the new PR #593 .